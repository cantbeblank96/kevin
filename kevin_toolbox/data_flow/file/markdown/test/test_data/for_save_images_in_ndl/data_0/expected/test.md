| read by | cv2 | torch | pil | else |
| --- | --- | --- | --- | --- |
| / | ![：cv2@0_.jpg](temp/：cv2@0_.jpg) | ![：torch@0_.jpg](temp/：torch@0_.jpg) | ![：pil@0_.png](temp/：pil@0_.png) | / |


