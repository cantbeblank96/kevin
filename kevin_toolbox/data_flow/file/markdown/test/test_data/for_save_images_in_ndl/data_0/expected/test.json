{
    "read by": [
        "image"
    ],
    "cv2": [
        "![：cv2@0_.jpg](plots/：cv2@0_.jpg)"
    ],
    "torch": [
        "![：torch@0_.jpg](plots/：torch@0_.jpg)"
    ],
    "pil": [
        "![：pil@0_.png](plots/：pil@0_.png)"
    ],
    "else": [
        "/"
    ]
}